### PR TITLE
Fix eas cloud build part-1

### PR DESCRIPTION
**Pull request by Tadjaur**

### CODE REVIEW

File: packages/app/envs/index.native.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Constants from 'expo-constants';
```
Comment (andrew-bierman) [inline packages/app/envs/index.native.ts:1]: can we handle this with the packages/config that we have ?

Comment (Tadjaur) [reply packages/app/envs/index.native.ts:1]: We should then have to create the `.env` file with all variable prefixed by `EXPO_PUBLIC_`.

Comment (Tadjaur) [reply packages/app/envs/index.native.ts:1]: Here is the list of update I plan to do:
- [ ] update imports from `@env` or `app/env` to `@packrat/config`
- [ ] update the workflow `android-preview-build-local.yml` to defined these variables
- [ ] update the readme

Comment (andrew-bierman) [reply packages/app/envs/index.native.ts]: @Tadjaur see the recent changes to the github action, can we map over these to set the env variables or will we have to call export manually for each one?

File: packages/app/components/map/WebMap.web.tsx [hunk 2]
```diff
@@ -1,15 +1,12 @@
 import React, { useState } from 'react';
 import mapboxgl from 'mapbox-gl';
-import { MAPBOX_ACCESS_TOKEN } from '@env';
```
Comment (andrew-bierman) [inline packages/app/components/map/WebMap.web.tsx:3]: we should use this instead of hardcoded key below, did not realize that was in there lol

**Pull request merged**